mangaki/mangaki/views.py

```diff
@@ -696,13 +694,6 @@ def add_pairing(request, artist_id, work_id):
     return HttpResponse()
 
 
-@receiver(user_signed_up)
-@receiver(social_account_added)
```

Comment:
Do we even support the ability to link new social accounts after being registered on Mangaki?